src/cpp/src/generation_config.cpp

```diff
@@ -222,7 +222,6 @@ void GenerationConfig::validate() const {
     }
 
     if (is_multinomial()) {
-        OPENVINO_ASSERT(top_k >= 0, "When 'do_sample' is true, top_k must be a non-negative, but got ", top_k);
```

Comment:
Please, change `static constexpr ov::Property<int> top_k{"top_k"};` to `size_t` to align them

Reply:
done